modules/@apostrophecms/page/index.js

```diff
@@ -481,6 +481,23 @@ module.exports = {
         }
       },
       'apostrophe:modulesReady': {
+        validateTypeChoices() {
+          for (const choice of self.typeChoices) {
+            if (!choice.name) {
+              throw new Error('One of the page types specified for your types option has no name property.');
+            }
+            if (!choice.label) {
+              throw new Error('One of the page types specified for your types option has no label property.');
+            }
+            if (!self.apos.modules[choice.name]) {
+              let error = `There is no module named ${choice.name}, but it is configured as a page type\nin your types option.`;
+              if (choice.name === 'home-page') {
+                error += '\n\nYou probably meant @apostrophecms/home-page.';
```

Comment:
I don't think we need this. If they're coming directly from 2.x the type would be `home`, so it's not helping that case.

Reply:
This was the bug in a3-assembly-boilerplate. I think it's an easier mistake for an A3 native to make because all the other page types look like `foo-type`. So I'd like to keep it.

Reply:
We'll want to deal with A2 migration in the A2 migration tool one day.